pkg/model/request_entity_test.go

```diff
@@ -18,7 +18,7 @@ package model
 import (
 	"fmt"
 	"testing"
-	"time"
+	//"time"
```

Comment:
Delete.

Reply:
Deleted it in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit